rasa/importers/autoconfig.py

```diff
@@ -0,0 +1,241 @@
+import copy
+import logging
+import os
+from pathlib import Path
+import tempfile
+
+from typing import Text, Dict, Any, List
+
+from rasa.cli import utils as cli_utils
+from rasa.constants import (
+    CONFIG_AUTOCONFIGURABLE_KEYS,
+    DOCS_URL_PIPELINE,
+    DOCS_URL_POLICIES,
+    CONFIG_KEYS,
+)
+from rasa.utils import io as io_utils, common as common_utils
+
+logger = logging.getLogger(__name__)
+
+COMMENTS_FOR_KEYS = {
+    "pipeline": (
+        f"# # No configuration for the NLU pipeline was provided. The following "
```

Comment:
@erohmensing Did you already double check the comments?

Reply:
thanks, look good but left one comment